cedar-policy-core/src/evaluator/err.rs

```diff
@@ -83,24 +84,25 @@ pub enum EvaluationError {
     InvalidRestrictedExpression(#[from] RestrictedExpressionError),
 
     /// Thrown when a policy is evaluated with an un-filled slot
-    #[error("Template slot {0} was not instantiated")]
+    #[error("template slot `{0}` was not filled")]
```

Comment:
I think "linked" is the correct terminology to use in the comment and error message.